### ISSUE TITLE
Define requirements to replace Calypso "Plugins" with WordPress "Plugins"

### ISSUE THREAD

**Issue by annezazu** [Automattic/wp-calypso#95652]
As part of the Untangling Calypso epic, this seeks to define requirements and any blockers for replacing wordpress.com/plugins/ view with the standard WordPress wp-admin/plugins.php view for new users.

This will reduce confusion navigating between Calypso and proper WordPress admin, creating a more coherent and expected experience.

### Current known differences

- Sorting/filtering differences: Calypso offers plugins defined by verticals both in sections as you scroll down and in tabs at the top whereas Core offers Featured, Popular, Recommended, Favorites.
- Upgrade nudges throughout Calypso.
- Distinct and contained plugin preview pages with nudges to upgrade when necessary.

Please edit to add more! This is just a start.

**Comment by Copons** [comment]
The distinction between Simple and Atomic is purely technical: Simple doesn't support plugins, so it doesn't need the "Installed Plugins" screen.

IMHO, we can keep the Marketplace as a separate item for the time being; I don't think there's much confusion.
In other words, I think we can close this as not planned. 🤔

**Comment by tanjoymor** [comment]
I agree with @Copons if there's no issue in keeping the Dotcom Plugin Marketplace as a separate item for now.

**Observations**
The order of the subitems: **Add new** and **Installed** are different on Classic vs Default - would be good to make this the same

Plugins > Installed
* Atomic Classic & Default UI load the same WP Admin view
* Simple/Classic loads an upsell screen
* Simple/Default n/a

Plugins > Add new
* Inconsistent behavior to be explored
* Atomic/Classic loads WP Admin Dotorg repo - Dotcom Marketplace is a separate link
* Atomic/Default loads Dotcom Marketplace - no access to Dotorg repo
* Simple/Classic loads an upsell screen
* Simple/Default n/a
